src/main/scala/org/apache/spark/sql/delta/storage/MemoryLogStore.scala

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2019 Databricks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.delta.storage
+
+import java.util.concurrent.{ConcurrentHashMap, TimeUnit}
+
+import com.google.common.cache.CacheBuilder
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.spark.SparkConf
+
+import collection.JavaConverters._
+import scala.util.control.Breaks._
+
+
+class MemoryLogStore(sparkConf: SparkConf, hadoopConf: Configuration)
```

Comment:
If this implementation only for tests, it should be probably moved out of here (but then it will be problem with loading by test suite, right?)